npm-shrinkwrap.json

```diff
@@ -2679,6 +2756,11 @@
       "version": "3.1.3",
       "resolved": "https://registry.npmjs.org/hawk/-/hawk-3.1.3.tgz"
     },
+    "he": {
+      "version": "1.1.1",
+      "resolved": "https://registry.npmjs.org/he/-/he-1.1.1.tgz",
```

Comment:
LOL, almost looks like "helll", which is a pretty good description of the shrinkwrap file :rofl: